pkg/environment/utils.go

```diff
@@ -60,3 +75,31 @@ func GetResolvedChannel(channel, inputChan, outputChan string) (string, error) {
 		Message("channel " + channel + " not listed as an input or output").
 		Build()
 }
+
+func getChannelData(channelList string) []brokers.ChannelBroker {
```

Comment:
Suggestion: Change this function name to something more descriptive, like getChannelsBrokerStructure.

Reply:
I left it as Data because this structure is soon going to be renamed and contain more information.